### PR TITLE
adding error msg, fixing footer

### PR DESCRIPTION
**Pull request by nisanthyaganti9**
https://www.pivotaltracker.com/story/show/184417979
https://www.pivotaltracker.com/story/show/184433221

### CODE REVIEW

File: libs/slcsp-calculator/premium-tax-credit/results/src/lib/results/results.component.ts [hunk 1]
```diff
@@ -42,6 +44,12 @@ export class ResultsComponent {
 
   @ViewChild('results', { static: false }) results!: ElementRef<HTMLDivElement>;
 
+  ngOnInit() {
+    setTimeout(() => {
+      this.canShowErrorMsg$.next(true);
+    }, 500);
+  }
+
```
Comment (TreyE) [inline libs/slcsp-calculator/premium-tax-credit/results/src/lib/results/results.component.ts:56]: Unless I'm misunderstanding how this is implemented, it simply shows  an error message after 500ms.  Doesn't this mean if the backend service takes longer than 500ms to come back we'll end up with a flash of the error message followed by showing the result?

**Review by TreyE — CHANGES_REQUESTED** [review]
Please see comments.

File: libs/slcsp-calculator/premium-tax-credit/results/src/lib/results/results.component.ts (hunk 1 above)
Comment (nisanthyaganti9) [reply libs/slcsp-calculator/premium-tax-credit/results/src/lib/results/results.component.ts:56]: right. when i checked a few scenarios, i've received response in < 500ms. But, i found another approach by adding `error` clause which removed dependency on setTimeOut.

**Review by polographer — APPROVED** [review]

File: libs/slcsp-calculator/premium-tax-credit/results/src/lib/results/results.component.ts (hunk 1 above)
Comment (TreyE) [reply libs/slcsp-calculator/premium-tax-credit/results/src/lib/results/results.component.ts:56]: Perfect.

**Review by TreyE — APPROVED** [review]

**Pull request merged**